source/docs/software/driverstation/driver-station-labview-dashboard.rst

```diff
@@ -0,0 +1,127 @@
+
+#. Camera Image Display
+#. Mode Selector - This drop-down allows you to select the type of camera display to use. The choices are Camera Off, USB Camera SW (software compression), USB Camera HW (hardware compression) and IP Camera (Axis camera). Note that the IP Camera setting will not work when your PC is connected to the roboRIO over USB.
+#. Camera Settings - This control allows you to change the resolution, framerate and compression of the image stream to the dashboard, click the control to pop-up the configuration.
+#. Bandwidth Indicator - Indicates approximate bandwidth usage of the image stream. The indicator will display green for "safe" bandwidth usage, yellow when teams should use caution and red if the stream bandwidth is beyond levels that will work on the competition field.
+#. Framerate - Indicates the approximate received framerate of the image stream.
+
+Drive
+-----
+
+.. image:: images/driver-station-labview-dashboard/drive.png
+
+The center pane contains a section that provides feedback on the joysticks and drive commands when used with the LabVIEW framework and a section that displays the robot IP and reported battery voltage:
+
+#. Displays X,Y and Throttle information and button values for up to 2 joysticks when using the LabVIEW framework
+#. Displays values being sent to motor controllers when using LabVIEW framework
+#. Displays a brief chart of the reported battery voltage
+#. Displays a Gyro value
+
+These indicators (other than the Gyro) are hooked up to appropriate values by default when using the LabVIEW framework. For information on using them with C++/Java code see :doc:`using-the-labview-dashboard-with-c++-java-code`.
+
+Autonomous Selection Box and Connection Indicator
+-------------------------------------------------
+
+.. image:: images/driver-station-labview-dashboard/autonomous-selection-box-and-connection-indicator.png
+
+This year, the Dashboard also contains 2 new elements on the Drive tab:
+
+#. A text box that can be used to select Autonomous modes. Each language's code templates have examples of using this box to select from multiple autonomous programs.
+#. A connection indicator for the connection to the robot.
+
+Basic
+-----
+
+.. image:: images/driver-station-labview-dashboard/basic.png
+
+The Basic tab contains a variety of pre-populated bi-directional controls/indicators which can be used to control the robot or display information from the robot. The SmartDashboard key names associated with each item are labeled next to the indicator with the exception of the Strings which follow the same naming pattern and increment from DB/String 0 to DB/String 4 on the left and DB/String 5 to DB/String 9 on the right. The LabVIEW framework contains an example of reading from the Buttons and Sliders in Teleop. It also contains an example of customizing the labels in Begin. For more detail on using this tab with C++\Java code see :doc:`using-the-labview-dashboard-with-c++-java-code`.
```

Comment:
Update for 2020 dashboard

Reply:
My version of the 2020 dashboard looks identical to the description. Is there a change I should be noting?

Reply:
I had misread the description